app/presenters/worldwide_organisation_presenter.rb

```diff
@@ -16,8 +16,13 @@ def sponsoring_organisation_links
   def world_location_links
     return if world_locations.empty?
 
+    world_location_name_translations = content_item.dig("details", "world_location_names")
+
     links = world_locations.map do |location|
-      link_to(location["title"], WorldLocationBasePath.for(location), class: "govuk-link")
+      world_location_translation = world_location_name_translations.select do |translation|
```

Comment:
Could you do a find here so you don't have to call .first?

Reply:
Yes, good idea.  I'd forgotten you could do `find` on an array.

Reply:
There's a failing test, but that will pass once the schema example update in https://github.com/alphagov/publishing-api/pull/2405 is merged.